http/src/v0/refs/path.rs

```diff
@@ -34,6 +34,9 @@ pub struct IpfsPath {
     /// Option to support moving the cid
     root: Option<Cid>,
     path: std::vec::IntoIter<String>,
+    /// True by default, to allow "finding" `Data` under dag-pb node
+    /// TODO: document why this matters
```

Comment:
just making sure this was not intended to be explained in this PR yet

Reply:
I think this can leave for now. Added note in the PR description under *Later*. This should be resolved during the later "move stuff from http to ipfs", like the `resolve` which this `IpfsPath` relates to. For my future self: I don't think I know about all of the ways the path resolution can happen. This value is however currently set everywhere it is cared for (both true and false).